### DISCUSSION TITLE
What's the difference between type of device options?

### DISCUSSION THREAD

**Discussion by Brix1330** [make-all/tuya-local#612] [Q&A]
For instance there's two choices when I add a smart plug: _smartplugv2_ and _smartplugv2_polled_power_.

**Comment by make-all** [comment]
The second one will poll the inputs for power/current/voltage every 30s.

Some smartplugs do not update those values automatically, so need to poll with a special command to force updates.
Other smartplugs do update automatically, and don't like the command to force updates (going offline/rebooting if it is given).

**Comment by make-all** [comment]
Try the `smartplugv2` config first.  If you see the power / current react within a few seconds of changing the load, then that will be the best config to use.  If it takes 5, 10, 30 minutes, or you never see a change, then you probably need `smartplugv2_polled_power`, so delete your device and set it up again with that config.
